extensions/platform-http/runtime/src/main/java/org/apache/camel/quarkus/component/platform/http/runtime/QuarkusPlatformHttpConsumer.java

```diff
@@ -336,19 +341,16 @@ static void populateCamelHeaders(
                     }
                 }
             }
-        }
 
-        // TODO: figure out whether we need this or remove
-        // // Create headers for REST path placeholder variables
-        // Map<String, Object> predicateContextParams = httpExchange.getAttachment(Predicate.PREDICATE_CONTEXT);
-        // if (predicateContextParams != null) {
-        // // Remove this as it's an unwanted artifact of our Undertow predicate chain
-        // predicateContextParams.remove("remaining");
-        //
-        // for (String paramName : predicateContextParams.keySet()) {
-        // headersMap.put(paramName, predicateContextParams.get(paramName));
-        // }
-        // }
+            /* Path parameters */
+            for (Entry<String, String> en : ctx.pathParams().entrySet()) {
+                final String name = en.getKey();
+                final String value = en.getValue();
+                if (!headerFilterStrategy.applyFilterToExternalHeaders(name, value, exchange)) {
```

Comment:
@davsclaus do we actually need to filter the path params? Maybe not, because the keys are defined by developer, not by the app user.

Reply:
Yeah if they are http context-path parameters then I dont see a reason for letting the camel filter strategy have a chance to filter them out.

Reply:
OK, let me remove the filtering. It is not there in the Undertow component too.